ruby/ql/src/queries/modeling/internal/Summaries.qll

```diff
@@ -0,0 +1,73 @@
+/**
+ * Contains predicates for generating `summaryModel`s to summarize flow through methods.
+ */
+
+private import ruby
+private import codeql.ruby.ApiGraphs
+private import codeql.ruby.TaintTracking
+private import Util as Util
+
+/**
+ * Contains predicates for generating `summaryModel`s to summarize flow through methods.
+ */
+module Summaries {
+  private module Config implements DataFlow::ConfigSig {
+    predicate isSource(DataFlow::Node source) { source instanceof DataFlow::ParameterNode }
```

Comment:
Should we restrict this to parameters of public methods?

Reply:
Yep, makes sense.